internal/cli/down.go

```diff
@@ -100,14 +106,59 @@ func (c *downCmd) down(ctx context.Context, downDeps DownDeps, args []string) er
 	return nil
 }
 
+func deleteK8sManifests(ctx context.Context, manifests []model.Manifest, updateSettings model.UpdateSettings, downDeps DownDeps, deleteNamespaces bool) error {
+	nodes := []*dependencyNode{}
```

Comment:
the api here is a bit odd. why only sort when deleting kubernetes resources? seems like docker-compose would benefit from this as well.

can you factor this out into a function that sorts the list of manifests in reverse order and returns it, then sort the list before of deleteK8sManifests / delete dc manifests?

Reply:
I think I've addressed all your comments now. This is now a function `sortManifestsForDeletion` and I call it in the `down` function before k8s and dc deletion. I'm not 100% sure I understand the dc case, since it looks like it assumes there's a single dc project, so let me know if you'd like something different from what I just pushed.

Reply:
we're working on supporting multiple dc projects :)